### PR TITLE
LPS-101019 Add ability to view diffs and links to content inside of a change list

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+		return portletURL.toString();
+	}
+
+	@Override
+	public Class<JournalArticle> getModelClass() {
+		return JournalArticle.class;
+	}
+
+	@Override
+	public String getTypeName(Locale locale) {
```
Comment (Preston-Crary) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java]: Want this to return "Web Content" but translated for the locale.

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+		try {
+			httpServletRequest.setAttribute(
+				WebKeys.RESOURCE_BUNDLE_LOADER,
+				ResourceBundleLoaderUtil.
+					getResourceBundleLoaderByServletContextName(
+						servletContext.getServletContextName()));
+
+			requestDispatcher.include(httpServletRequest, httpServletResponse);
+		}
+		catch (ServletException se) {
```
Comment (Preston-Crary) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java]: No need to catch here, method throws exception.

Comment (Preston-Crary) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java]: Don't need to log either.

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+		PortletPreferences portletPreferences = portletRequest.getPreferences();
+
+		String languageId = LanguageUtil.getLanguageId(httpServletRequest);
+		int page = ParamUtil.getInteger(httpServletRequest, "page", 1);
+		String viewMode = ParamUtil.getString(httpServletRequest, "viewMode");
+
+		String ddmTemplateKey = PrefsParamUtil.getString(
+			portletPreferences, httpServletRequest, "ddmTemplateKey");
+
+		if (Validator.isNull(ddmTemplateKey)) {
```
Comment (Preston-Crary) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/display/JournalArticleCTDisplayRenderer.java]: Pretty sure ddmTemplateKey is always null.

**Comment by stsquared99** [comment]
@Preston-Crary Updated

**Comment by Preston-Crary** [comment]
https://github.com/stsquared99/liferay-portal/pull/79

**Pull request closed without merging**